### PR TITLE
[Performance](exec) opt the exchange performance

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

File: be/src/agent/be_exec_version_manager.h [hunk 1]
```diff
@@ -64,7 +64,7 @@ class BeExecVersionManager {
  *    c. cleared old version of Version 2.
  *    d. unix_timestamp function support timestamp with float for datetimev2, and change nullable mode.
 */
-inline const int BeExecVersionManager::max_be_exec_version = 3;
+inline const int BeExecVersionManager::max_be_exec_version = 4;
```
Comment (zclllyybb) [inline be/src/agent/be_exec_version_manager.h:67]: there should be a version change note up here

**Comment by HappenLee** [comment]
run buildall

**Comment by hello-stephen** [comment]
run performance

**Comment by HappenLee** [comment]
run buildall

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Review by BiteTheDDDDt — APPROVED** [review]

**Pull request merged**